### ISSUE TITLE
Consider putting all MsftBloat in separate script file

### ISSUE THREAD

**Issue by eabase** [Disassembler0/Win10-Initial-Setup-Script#350]
It would be helpful and considerably easier to maintain, if we were to put all the MS bloat tweak into it's own `*.ps1` file. In this way, if the user has `UninstallMsftBloat` enabled, then that file would be run...

**Comment by spaceneenja** [comment]
Agree, also a separate editable .preset file would be useful for this section as well.
